whereby/sdk
# Add spotlight functionality

### Description
Adds spotlighting functionality:
* `spotlightParticipant` fn in the useRoomConnection actions
* `removeSpotlight` fn in the useRoomConnection actions
* Grid respects spotlights, and adds all spotlighted participants to the presentation grid

**Summary:**



**Related Issue:**



### Testing
1. `yarn build && yarn dev`
2. Open the storybook story: "Custom UI" > "Room Connection With Host Controls" (with a host key configured in the .env file)
3. Join the same room in a separate tab (with the PWA)
4. Click the "spotlight" button on one of the participants
5. Verify that the participant is spotlighted (blue border in storybook, spotlighted in the PWA)
6. Click the remove spotlight button on the spotlighted participant
7. Verify that the spotlight is removed (both tabs)
8. Go to the grid story
9. Spotlight a participant (from the PWA)
10. Verify that the spotlighted participant is shown in the presentation grid (bigger cell)
11. You can play around with more participants and spotlighting/removing spotlights

### Screenshots/GIFs (if applicable)



### Checklist

        under the terms of the MIT license.

### Additional Information
